invenio/modules/communities/templates/communities/new_base.html

```diff
@@ -214,5 +220,74 @@ <h3 id="deletemodal_label">Delete community?</h3>
 
 init_ckeditor("page", 'advanced');
 init_ckeditor("description", 'simple');
+
+/**
+ * Module validating FileField
+ * @module FileFieldValidator
+ */
+(function () {
+    "use strict";
+    /** @constant {number}
+     *  @default 1.5MB
+     */
+    var MAX_SIZE = 1.5e+6;
+    /** @constant {string[]} */
+    var COMMUNITIES_LOGO_EXTENSIONS = ['.png', '.jpg', '.jpeg']
```

Comment:
you probably want to write something like `var COMMUNITIES_LOGO_EXTENSIONS = {{ config.COMMUNITIES_LOGO_EXTENSIONS|tojson}};`